### PR TITLE
[REF] default_warehouse_from_sale_team: follow guidelines to clean the code T#63809

### PR DESCRIPTION
**Pull request by rolandojduartem**

### CODE REVIEW

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, please, could you review this PR?

cc @desdelinux

File: default_warehouse_from_sale_team/demo/crm_team_demo.xml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+<?xml version='1.0' encoding='UTF-8' ?>
+<odoo>
+
+    <record id="stock_warehouse_default_team" model="stock.warehouse">
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/demo/crm_team_demo.xml]: It seems records are duplicated across files

File: default_warehouse_from_sale_team/security/ir_rule_security.xml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+            eval="[(6, 0, [ref('default_warehouse_from_sale_team.group_limited_default_warehouse_spt')])]"
+        />
+    </record>
+    <record id="rule_default_warehouse_stock_picking_type_all" model="ir.rule">
+        <field name="name">Access to all picking types</field>
+        <field name="model_id" search="[('model','=','stock.picking.type')]" model="ir.model" />
+        <field name="domain_force">[(1, '=', 1)]</field>
+        <field
+            name="groups"
+            eval="[(6, 0, [ref('default_warehouse_from_sale_team.group_manager_default_warehouse_spt')])]"
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/security/ir_rule_security.xml]: - command syntax (separated commit)
- Superfluous module in external ID (look for other possible cases)

File: default_warehouse_from_sale_team/security/ir_rule_security.xml [hunk 3]
```diff
@@ -0,0 +1,106 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo noupdate="0">
```
Comment (luisg123v) [inline default_warehouse_from_sale_team/security/ir_rule_security.xml]: Superfluous noupdate 0 (separated commit).

Same in similar cases

**Review by luisg123v — CHANGES_REQUESTED** [review]

**Comment by rolandojduartem** [comment]
Hi, @luisg123v, could you review this PR? The red CI is not related to these changes.

**Review by luisg123v — APPROVED** [review]

**Pull request merged**